README.md

```diff
@@ -296,7 +296,10 @@ labels:
   incremental_export: false  # non-incremental JSON export writes all data to a single location
 ```
 
-### Publishing Datasets
+### Publishing a Table Publicly
+
+For background, see [Accessing Public Data](https://docs.telemetry.mozilla.org/cookbooks/public_data.html)
+on `docs.telemetry.mozilla.org`.
```

Comment:
Unrelated change for something I was confused by while reading through this file.